vignettes/imputation.Rmd

```diff
@@ -106,8 +111,9 @@ dates <- c(
   "2019",
   "2019---01"
 )
-impute_dtc(
+impute_dtc_dtm(
   dates,
+  highest_imputation = "M",
   date_imputation = "06-15",
   time_imputation = "00:00:00"
 )
```

Comment:
I would add a couple of examples of different values of `highest_imputation` here now, e.g. imputing only minutes/seconds or imputing up to days but not months.

Reply:
I have added a subsection for the imputation level.